experiments/machxo2/ccu2_mux/ccu2_diff.txt

```diff
@@ -0,0 +1,322 @@
+CENTER10:CENTER_DUMMY
```

Comment:
Is this a file that needs to be committed into the repo? It looks more like some kind of output.